controllers/articles.controllers.js

```diff
@@ -11,12 +11,28 @@ exports.getArticleById = (req, res, next) => {
 }
 
 exports.getAllArticles = (req, res, next) => {
-    fetchAllArticles().then((articles) => {
-        res.status(200).send({ articles })
-    })
-    .catch((err) => {
-        next(err)
-    })
+    const { topic } = req.query;
+    const validTopicQueries = ['cats', 'mitch']
```

Comment:
We have another valid topic available, but in general we don't want to hard code these values otherwise we could have issues later if we added the functionality to create more topics.